.github/workflows/test-action.yml

```diff
@@ -0,0 +1,55 @@
+---
+name: Test ansible-test-gh-action
+on:
+  push:
+    branches: [main]
```

Comment:
In this repository, `main` is not the only important branch. Plus, since I'm the only one pushing directly (will probably stop soon anyway), limiting CI branches does more harm than good. So I'm going to drop this for now.
```suggestion
```